### ISSUE TITLE
Update question images

### ISSUE THREAD

**Issue by cgray1117** [EducationShinyAppTeam/Time_Series_Models#1]
The images for questions 20, 21, 36, and 37 in the tic tac toe question bank should be reviewed and improved for better accessibility.

**Comment by shravanisamala** [comment]
Tic-tac- toe game was removed so this can be something we look into for the game app
